### PR TITLE
Refacto:  coverage

### PR DESCRIPTION
**Pull request by kevbarns**

### CODE REVIEW

**Review by ericplaquevent — APPROVED** [review]

File: server/src/jobs/psCoverage/pscoverage.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    };
+
+    for (let i = 0; i < cfd.length; i++) {
+      let { query, strengh } = cfd[i];
+
+      let result = await getMatch(query(formation));
+
+      if (result.length > 0) {
+        buffer.match.push({
+          matching_strengh: strengh,
```
Comment (ericplaquevent) [inline server/src/jobs/psCoverage/pscoverage.js:31]: Probably nothing, but you have a typo : strengh --> **strength**

Comment (kevbarns) [reply server/src/jobs/psCoverage/pscoverage.js:31]: Yes bien vu, vilaine typo dans tout les fichiers

File: server/src/jobs/psCoverage/pscoverage.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            matching_strengh: strengh,
+            data_length: result.length,
+            data: formatFormation(result),
+          });
+
+          break;
+        }
+      }
+    }
+
```
Comment (kevbarns) [inline server/src/jobs/psCoverage/pscoverage.js:57]: if buffer is empty, return

File: server/src/jobs/psCoverage/queries.js [hunk 3]
```diff
@@ -0,0 +1,232 @@
+const uai = [
```
Comment (kevbarns) [inline server/src/jobs/psCoverage/queries.js:1]: How would this code could be better implemented ? @ericplaquevent @Gethi

**Pull request merged**